### DISCUSSION TITLE
Enhancements and focus on frontend development

### DISCUSSION THREAD

**Discussion by MDeLuise** [MDeLuise/plant-it#136] [Announcements]
Hi Plant-it community!
I wanted to take a moment to share with you some updates regarding the upcoming project release.

Throughout the course of the project, I've noticed a recurring issue with deployment - many users have encountered difficulties connecting to the backend, leading to numerous support tickets. While I've made efforts to enhance the documentation with clearer instructions and helpful examples, I believe there's room for improvement.

Additionally, it's become apparent that the frontend requires significant redesign and restructuring. As mentioned in [this issue](https://github.com/MDeLuise/plant-it/issues/113), the current layout lacks responsiveness and falls short on larger screens.

To address these challenges, I've decided to prioritize the frontend of the project. My plan is to develop a standalone application that users can install on their devices, which will seamlessly connect to one backend service instance. By removing the frontend service from the deployment, I'll simplify the process and hopefully eliminate the connectivity issues.

However, I still recognize the importance of offering a web application for those who prefer it. Moving forward, users will have the option to choose between accessing the service via the web or through a downloadable app.

Initially, I was leaning towards using React Native for the app development. However, after receiving feedback from some of you on Reddit, I've decided to explore Flutter as well. Before diving into development, I'll be conducting some initial exploration to assess which platform best suits our needs.

Although the backend could accommodate numerous additional features, I believe it has  attained a robust initial feature set. Therefore, I think it's the right time to prioritize the frontend before proceeding with further backend enhancements.

Stay tuned for further updates as I progress and happy planting 🪴

**Comment by Samurai1201** [comment]
Very happy, that a web version will still be available to use :)
Thanks for your dedication, looking forward to seeing the future development

**Comment by MDeLuise** [comment]
I'm closing this discussion since the new webapp is available in the project version `0.5.0`.
For next steps of the project please see https://github.com/MDeLuise/plant-it/discussions/147.
